docs/developer.rst

```diff
@@ -192,7 +192,8 @@ documentation/developer
 category
     * optional
     * if not provided the category **tools** will be used
-    * must contain one of the following values: **customization**, **files**, **integration**, **monitoring**, **multimedia**, **office**, **organization**, **social**, **tools**
+    * must contain one of the following values: **auth**, **customization**, **files**, **integration**, **monitoring**, **multimedia**, **office**, **organization**, **social**, **tools**
+    * old categories are migrated (however: duplicate result categories are invalid!): **tool**, **game**, **other** -> **tools**, **productivity** -> **organization**
```

Comment:
It seems a bit unclear to me what categories are migrated to what. I would make one sub list item for each "->". Also, I have not really grasped what it is that is invalid :)